### ISSUE TITLE
Forbid only and forbid pending

### ISSUE THREAD

**Issue by nathanjisaac** [zinserjan/mocha-webpack#197]
The mocha cli offers the following options. Is there a way to add this to `mocha-weback`? 

https://mochajs.org/#usage

```
    --forbid-only                           causes test marked with only to fail the suite
    --forbid-pending                        causes pending tests and test marked with skip to fail the suite
```

It would be nice to run a command like so: 

```
mocha-webpack --forbid-only --forbid-pending
```

**Comment by nathanjisaac** [comment]
I just found this as another approach to this feature. 

https://github.com/lo1tuma/eslint-plugin-mocha

**Comment by nathanjisaac** [comment]
My guess is that this is best solved using a linter.

**Comment by Janpot** [comment]
@nathanjisaac I don't think it should be solved by the linter. This is a mocha specific option and I only want it when I run my tests in a CI server.
I think it makes most sense that `mocha-webpack` should just passes through all of the options to `mocha`.
Would you mind reopening this?

**Comment by nathanjisaac** [comment]
Sure, I can reopen this issue.

**Comment by zinserjan** [comment]
I'm happy to receive a PR for this ;)

**Comment by Janpot** [comment]
I still think it makes sense in the context of this library. But I've moved on to a webpack+mocha approach that makes more sense to me. So I'm not actively using this library anymore. Feel free to close this issue again if you want.

**Comment by JamesMcMahon** [comment]
@zinserjan This is a bit old at this point but I was thinking of working on PR for this.

Would a better approach be allowing unknown flags to be deferred down to mocha? IE, take a blacklist approach vs whitelisting certain flags.
